science/orthanc-dicomweb/Makefile

```diff
@@ -0,0 +1,64 @@
+# Created by: maintainer.freebsd@xpoundit.com
+
+PORTNAME=	orthanc-dicomweb
+DISTVERSION=	1.6
+CATEGORIES=	science
+MASTER_SITES=	https://www.orthanc-server.com/downloads/get.php?path=/plugin-dicom-web/:main \
+		https://www.orthanc-server.com/downloads/get.php?path=/orthanc/:framework \
+		https://orthanc.osimis.io/ThirdPartyDownloads/:thirdparty \
+		https://orthanc.osimis.io/ThirdPartyDownloads/dicom-web/:thirdpartydicomweb
+DISTFILES=	OrthancDicomWeb-${PORTVERSION}.tar.gz:main \
+		Orthanc-1.9.3.tar.gz:framework \
+		e2fsprogs-1.44.5.tar.gz:thirdparty \
+		bootstrap-4.3.1.zip:thirdpartydicomweb \
+		vuejs-2.6.10.tar.gz:thirdpartydicomweb \
+		axios-0.19.0.tar.gz:thirdpartydicomweb \
+		Font-Awesome-4.7.0.tar.gz:thirdpartydicomweb \
+		bootstrap-vue-2.0.0-rc.24-dist.tar.gz:thirdpartydicomweb \
+		babel-polyfill-6.26.0.min.js.gz:thirdpartydicomweb
+DIST_SUBDIR=	orthanc
+EXTRACT_ONLY=	OrthancDicomWeb-${PORTVERSION}.tar.gz
+
+MAINTAINER=	maintainer.freebsd@xpoundit.com
+COMMENT=	Plugin to bring support of the DICOMweb standard into Orthanc
+
+LICENSE=	AGPLv3
+LICENSE_FILE=	${WRKSRC}/COPYING
+
+BUILD_DEPENDS=	${LOCALBASE}/include/orthanc/OrthancCPlugin.h:science/orthanc
+LIB_DEPENDS=	libboost_atomic.so:devel/boost-libs \
+		libexpat.so:textproc/expat2 \
+		libgdcmCommon.so:devel/gdcm \
+		libicudata.so:devel/icu \
+		libjsoncpp.so:devel/jsoncpp \
+		libopenjp2.so:graphics/openjpeg \
+		libpugixml.so:textproc/pugixml
+RUN_DEPENDS=	Orthanc:science/orthanc
+
+USES=		cmake localbase python:build
+USE_LDCONFIG=	yes
+
+WRKSRC=		${WRKDIR}/OrthancDicomWeb-${PORTVERSION}
+
+CMAKE_OFF=	USE_SYSTEM_UUID
+CMAKE_ARGS=	-DORTHANC_FRAMEWORK_ROOT=${WRKSRC}/ThirdPartyDownloads/Orthanc-1.9.3/OrthancFramework/Sources \
+		-DORTHANC_FRAMEWORK_SOURCE=path
+CFLAGS+=	-DORTHANC_ENABLE_LOGGING_PLUGIN -DNDEBUG
+CXXFLAGS+=	-DNDEBUG
+
+post-extract:
+		${MKDIR} ${WRKSRC}/ThirdPartyDownloads
+		${CP} ${DISTDIR}/${DIST_SUBDIR}/e2fsprogs-1.44.5.tar.gz ${WRKSRC}/ThirdPartyDownloads
+		${CP} ${DISTDIR}/${DIST_SUBDIR}/bootstrap-4.3.1.zip ${WRKSRC}/ThirdPartyDownloads
+		${CP} ${DISTDIR}/${DIST_SUBDIR}/vuejs-2.6.10.tar.gz ${WRKSRC}/ThirdPartyDownloads
+		${CP} ${DISTDIR}/${DIST_SUBDIR}/axios-0.19.0.tar.gz ${WRKSRC}/ThirdPartyDownloads
+		${CP} ${DISTDIR}/${DIST_SUBDIR}/Font-Awesome-4.7.0.tar.gz ${WRKSRC}/ThirdPartyDownloads
+		${CP} ${DISTDIR}/${DIST_SUBDIR}/bootstrap-vue-2.0.0-rc.24-dist.tar.gz ${WRKSRC}/ThirdPartyDownloads
+		${CP} ${DISTDIR}/${DIST_SUBDIR}/babel-polyfill-6.26.0.min.js.gz ${WRKSRC}/ThirdPartyDownloads
+		${CP} ${DISTDIR}/${DIST_SUBDIR}/Orthanc-1.9.3.tar.gz ${WRKSRC}/ThirdPartyDownloads
```

Comment:
Also just nitpicking, if this was me, I would use a `.for` loop for `${DISTFILES}`